lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -46,6 +46,24 @@ function dosomething_user_init() {
       dosomething_user_non_staff_redirect();
     }
   }
+
+  // If viewing a user/[northstar id]
+  if(arg(0) == 'user') {
```

Comment:
@angaither it doesn't look like `dosomething_user_menu()` is getting called? No dpms from in there are printing out and when I move the code into there it no longer works